### DISCUSSION TITLE
1주차 포스팅 주제

### DISCUSSION THREAD

**Discussion by Ariling** [PotatoesStudy/FE-hub#4] [Announcements]
## 주제 : JS (+ TS)
**12/31일까지 주제 겹치지 않게 선착순으로 작성하기**
> [!IMPORTANT]
> **1개 이상 할 경우 필독**
> 1개를 먼저 포스팅 한 다음에 하기 -> 먼저 3개 올렸다가 추후에 감당 안 돼서 안하는 경우를 방지하기 위함

**Comment by Ariling** [comment]
비동기 정복하기 (JS의 비동기 처리 방식 + 동기)

**Comment by gominzip** [comment]
클로저 (feat. react hook)

**Comment by spearStr** [comment]
함수와 일급객체

**Comment by Gyeongbin** [comment]
JS의 이벤트 루프와 스케줄링(+Web worker)
